tools/run-wpt.mjs

```diff
@@ -135,11 +143,12 @@ if (CHROMEDRIVER === 'true') {
     '--binary-arg=--headless=new',
     '--install-webdriver',
     `--webdriver-arg=--bidi-mapper-path=${join('lib', 'iife', 'mapperTab.js')}`,
-    `--webdriver-arg=--log-path=${join('out', 'chromedriver.log')}`,
+    `--webdriver-arg=--log-path=${LOG_FILE}`,
     '--webdriver-arg=--verbose',
     '--yes'
   );
 } else {
+  wptRunArgs.push('--webdriver-binary', 'tools/run-bidi-server.mjs');
```

Comment:
Use `join` here, for compatibility with Windows (like bidi-mapper-path above)

Reply:
done